extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -253,79 +240,6 @@ function Ach_TakeRankingSnapshot($force = false) {
 	$dbw->freeResult($res);
 }
 
-##
-## TESTING ONLY
-##
-//$wgAjaxExportList[] = 'AchTest';
-function AchTest() {
-
-	global $wgUser, $wgExternalSharedDB;
-
-	set_time_limit(5000);
-
-		// test 1
-		// Expect: badges for edits, categories, pictures, at least one lucky edit and eventually welcome
-
-		// 2000 edits
-		for($i = 0; $i < 500; $i++) {
-
-			$content = date('H:i:s');
-
-			// 2000 category edits
-			$content .= '[[Category:cat'.$i.']]';
-			$content .= '[[Category:Moja testowa]]';
-
-			// 1000 picture uploads
-			if($i % 2 == 0) {
-				$content .= '[[Image:Wiki.png]]';
-			}
-
-			$title = Title::newFromText('CArticle_'.$i);
-			$article = new Article($title);
-			$article->doEdit($content, 'test summary');
-		}
-
-		$dbw = wfGetDB(DB_MASTER, array(), $wgExternalSharedDB);
-		$dbw->commit();
-
-		$dbw = wfGetDB(DB_MASTER);
-		$dbw->commit();
```

Comment:
It appears to have been only used for testing years ago and its `$wgAjaxExportList` entry is commented out: https://github.com/Wikia/app/pull/13444/files/3d502da2f23fceed6ea1eb05f5ed72a62edb0639#diff-f1fb8c62f74f81312278d6454394bcceL259